dashboard/app/controllers/projects_lists_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProjectsListsController < ApplicationController
```

Comment:
Let me answer your question by asking a question :) in the future, we will want to serve a list of projects that represents all the projects owned by a single student. this will be viewed by the student, not the teacher. should this controller be named so as to accommodate those requests also? or should that be handled by a different controller?